persist-cli-tests/src/test/resources/test-src/input/tool_test_build_1/Ballerina.toml

```diff
@@ -32,4 +32,4 @@ filePath = "persist/model.bal"
 [[platform.java21.dependency]]
 groupId = "io.ballerina.stdlib"
 artifactId = "persist.sql-native"
-version = "1.4.0"
+version = "1.4.1"
```

Comment:
is this expected to have version `1.4.1` or `1.5.0-SNAPSHOT`?

Reply:
Fixed in [0d19498](https://github.com/ballerina-platform/persist-tools/pull/391/commits/0d1949880afb5ba7cf1949b564f711b368748d87)